modules/ROOT/pages/acm-lightning-components.adoc

```diff
@@ -56,6 +56,10 @@ The API carousel component shows a horizontally scrollable set of API cards.
 
 The interactive API console component shows information about your APIs and their methods, and allows users to test the methods by sending requests.
 
+Endpoints are displayed in alphabetical order, even when the original API specification listed them in a different order.
```

Comment:
Does the latter half of the sentence add value? Can we remove it and have 'Endpoints are displayed in alphabetical order.'?

Reply:
Yes, that is better. Removed the latter half.